packages/common/src/core/schemas/internal/metrics/interfaces.ts

```diff
@@ -0,0 +1,45 @@
+export enum LAYOUTS {
+  simple = "simple",
+  dataViz = "dataViz",
+  moreInfo = "moreInfo",
+}
+export enum SCALE {
+  small = "s",
+  medium = "m",
+  large = "l",
+}
+export enum UNIT_POSITIONS {
+  before = "before",
+  after = "after",
+  below = "below",
+}
+export enum VISUAL_INTEREST {
+  none = "none",
+  sparkline = "sparkline",
+  icon = "icon",
+  logo = "logo",
+}
+
+export enum SHARING {
+  always = "always",
+  hover = "hover",
+}
+
+export enum SOURCE {
+  dynamic = "dynamic",
+  static = "static",
+}
+
+export interface IShareableCard {
```

Comment:
still wondering if this and `IMAGE_TYPES` should be exported from `hub.js` at all. I don't think they're related to metrics?